rakelib/run_rspec.rake

```diff
@@ -0,0 +1,65 @@
+require "coveralls/rake/task"
+require "pathname"
+
+require_relative "task_helpers"
+require_relative "example_type"
+
+include ReactOnRails::TaskHelpers
+
+namespace :run_rspec do
+  desc "Run RSpec for top level only"
```

Comment:
Can we also lint the code, only the JS part? would that make sense? My concern is that we generate some JS that does not lint.